src/control/Control.cpp

```diff
@@ -1247,7 +1247,7 @@ void Control::appendNewPdfPages() {
                 return page->getPdfPageNr() + 1;
             }
         }
-        return 0ULL;
+        return 0UL;
```

Comment:
This is only correct on 32bit or Linux machines.
`return ULL;` was already correct, since we don't support 32 bit.
But you can support all machines / compiler:
```suggestion
        return size_t{0U};
```

Reply:
Thanks for the fix. It didn't compile with `return 0ULL` on my (64-bit Linux) machine (gcc 9.2.1). That's why I changed it. I am glad you checked it once more. With `size_t{0U}` it works.